### PR TITLE
REF: Add Wallet advanced options

### PR DESCRIPTION
**Pull request by marcosrdz**
Screenshot 2024-09-05 at 4 43 49 PM

## Summary by CodeRabbit

- **New Features**
	- Removed "Advanced Options" from the user interface to simplify settings.
	- Introduced a new `ToolTipMenu` for selecting wallet types and managing entropy generation.
	- Enhanced functionality of wallet management through updated tooltip actions.

- **User Interface Enhancements**
	- Updated terminology for "Vault" to "Multisig Vault" for clarity.
	- Improved responsiveness of the `WalletsAdd` component with optimized state management and layout logic.
	- Enhanced interactivity in wallet details and multisig components through new tooltip menus.
	- Streamlined navigation with a new close button position in the `ReceiveDetails` screen.

- **Bug Fixes**
	- Adjusted UI interaction sequences in tests to reflect changes in the application’s flow.

- **Chores**
	- Enhanced Fastlane configuration for more efficient branch management during iOS app deployments.
	- Updated GitHub Actions workflows for improved artifact handling and deployment processes.

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: components/TooltipMenu.tsx [hunk 1]
```diff
@@ -8,10 +9,103 @@ import {
...
+ *     borderRadius: 5,
+ *   },
+ *   buttonText: {
+ *     color: '#fff',
+ *   },
+ * });
+ *
+ * export default ExampleToolTipMenu;
+ * ```
+ */
```
Comment (Overtorment) [inline components/TooltipMenu.tsx:108]: dead code.
also why youre at it, can you make it not pollute console?

**Review by Overtorment — CHANGES_REQUESTED** [review]
mostly lgtm

File: components/TooltipMenu.tsx (hunk 1 above)
Comment (marcosrdz) [reply components/TooltipMenu.tsx:108]: Its example on how to use it. Pollution removed

**Pull request merged**